public/stylesheets/themes.css

```
@@ -0,0 +1,195 @@
+* {
+    font-family: monospace;
+}
+
+body {
+    background-color: #202020;
+    background: linear-gradient(135deg, #42280055 25%, transparent 25%) -13px 0/ 26px 26px, linear-gradient(225deg, #422800 25%, transparent 25%) -13px 0/ 26px 26px, linear-gradient(315deg, #42280055 25%, transparent 25%) 0px 0/ 26px 26px, linear-gradient(45deg, #422800 25%, #202020 25%) 0px 0/ 26px 26px;
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)